scripts/update_api_description.sh

```diff
@@ -17,7 +17,7 @@ function main() {
 
     mkdir -p "${OUT_DIR}"
 
-    go run cmd/describe-api/main.go > "${OUT_FILE}"
+    go run cmd/call-endpoint/main.go metadata/describe > "${OUT_FILE}"
```

Comment:
Just and FYI.
Hopefully github alerted you to this comment:
https://github.com/edulinq/autograder-server/pull/131#discussion_r1876974800